Improve setup documentation (#143)
swisstopo/swissgeol-boreholes-dataextraction: README.md

2. **Download the borehole profiles, optional**

Use `boreholes-download-profiles` to download the files to be processed from an AWS S3 storage. In order to do so, you need to authenticate with aws first. We recommend using the aws CLI for that purpose, or storing your credentials in the ~/.aws configuration files This step is optional, you can continue with step 3 on your own set of borehole profiles.

Alternativelly, you can download the data directly using the AWS CLI:
```bash 
brew install awscli
aws s3 sync s3://stijnvermeeren-boreholes-data ./data
```

> If you choose to use the ~/.aws files, here how they should look like.

Reviewer: ```suggestion
If you choose to use the ~/.aws files, then they should look like this:
```

- `main.py` : This is the main script of the project. It runs the data extraction pipeline, which analyzes the PDF files in the `data/Benchmark` directory and saves the results in the `predictions.json` file.

## Data

> To locally download the data from the AWS S3 bucket to the `./data` directory, run the following commands:

Reviewer: Isn't this already covered by the `boreholes-download-profiles` script that is documented higher up in this README?

Author: Yes, it missed it :) I moved my explanations on the right section. I can just remove them if you prefer, but it might be usefull to still have the method with the cli that I used.
